experimental/PropertyDDS/services/property-query-service/src/server/utils/request_utils.js

```diff
@@ -6,7 +6,7 @@
 	const _ = require("lodash");
 	const async = require("async");
 	const HTTPError = require("@fluid-experimental/property-common").HTTPErrorNoStack;
-	const request = require("request");
+	const axios = require("axios");
```

Comment:
Please make this visible to owners/devs working on PropertyTree

Reply:
This seems dangerous. Are there other options to fix it?